### PR TITLE
docs: rewrite intro

### PR DESCRIPTION
**Pull request by c-ehrlich**
## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

- improved docs intro
- no more nelson gif
- move the axioms into the intro page

I'd suggest to make similar changes to the main readme if this looks good to everyone else. Could also see going even further on rewriting a completely different intro, lmk - just wanted to open a PR so we get started discussing it :)

💯

### CODE REVIEW

File: www/src/pages/en/introduction.md [hunk 1]
```diff
@@ -33,13 +33,18 @@ The _"T3 Stack"_ is a web development stack made by [Theo](https://twitter.com/t
```
Comment (juliusmarminge) [inline www/src/pages/en/introduction.md:33]: Line 32:

What's `create-t3-app` then? Some kinda template?

Coming from the section above, "This" points towards the T3 Stack

File: www/src/pages/en/introduction.md [hunk 2]
```diff
@@ -33,13 +33,18 @@ The _"T3 Stack"_ is a web development stack made by [Theo](https://twitter.com/t
...
+
+It's easy to fall in the trap of "adding everything" - we explicitly don't want to do that. Everything added to `create-t3-app` should solve a specific problem that exists within the core technologies included. This means we won't add things like state libraries (`zustand`, `redux`) but we will add things like NextAuth.js and integrate Prisma and tRPC for you.
+
+### Bleed Responsibly
+
+We love our bleeding edge tech. The amount of speed and, honestly, fun that comes out of new shit is really cool. We think it's important to bleed responsibly, using riskier tech in the less risky parts. This means we wouldn't ⛔️ bet on risky new database tech (SQL is great!). But we happily ✅ bet on tRPC since it's just functions that are trivial to move off.
+
+### Typesafety Isn't Optional
+
+Two of the three T's are typesafe (Typescript, tRPC). We take typesafety seriously in these parts. Any decision that compromises the full-stack, typesafe nature of `create-t3-app` is a decision that should be made in a different project. The stated goal of `create-t3-app` is to provide the quickest way to start a new fullstack, type safe web application.
```
Comment (juliusmarminge) [inline www/src/pages/en/introduction.md:50]: Two of the three T's are regarding typesfety ...

Comment (nexxeln) [reply www/src/pages/en/introduction.md:50]: probably no apostrophe on the T too

**Review by nexxeln — COMMENTED** [review]
Mirror this on the Readme too

**Comment by AyanavaKarmakar** [comment]
The link should be updated to point towards `/introduction#typesafety-isnt-optional` instead of `/t3-axioms#typesafety-isnt-optional`, since the T3 Axiom page doesn't exist, and will lead to 404.

image

File: www/src/pages/en/introduction.md (hunk 1 above)
Comment (c-ehrlich) [reply www/src/pages/en/introduction.md:33]: Changed the heading, here and in readme

File: www/src/pages/en/introduction.md (hunk 2 above)
Comment (c-ehrlich) [reply www/src/pages/en/introduction.md]: rewrote this paragraph, here and in readme

File: www/src/pages/en/introduction.md [hunk 3]
```diff
@@ -47,4 +47,4 @@ We love our bleeding edge tech. The amount of speed and, honestly, fun that come
 
 ### Typesafety Isn't Optional
 
-Two of the three T's are typesafe (TypeScript, tRPC). We take typesafety seriously in these parts. Any decision that compromises the full-stack, typesafe nature of `create-t3-app` is a decision that should be made in a different project. The stated goal of `create-t3-app` is to provide the quickest way to start a new fullstack, type safe web application.
\ No newline at end of file
+The stated goal of `create-t3-app` is to provide the quickest way to start a new full-stack, typesafe web application. We take typesafety seriously in these parts as it improves our productivity and helps us ship fewer bugs. Any decision that compromises the typesafe nature of `create-t3-app` is a decision that should be made in a different project.
```
Comment (nexxeln) [inline www/src/pages/en/introduction.md]:
```suggestion
The stated goal of `create-t3-app` is to provide the quickest way to start a new full-stack, **typesafe** web application. We take typesafety seriously in these parts as it improves our productivity and helps us ship fewer bugs. Any decision that compromises the typesafe nature of `create-t3-app` is a decision that should be made in a different project.
```

nit but making it bold is nice here.

**Review by nexxeln — CHANGES_REQUESTED** [review]

File: www/src/pages/en/introduction.md [hunk 4]
```diff
@@ -29,17 +29,22 @@ The _"T3 Stack"_ is a web development stack made by [Theo](https://twitter.com/t
...
 
-<p align="center">
-  <img
-    src="https://media4.giphy.com/media/3orieLHXgpfkKO9Iju/200.gif"
-    alt="Nelson from the Simpsons pointing and saying ha-ha in a mocking tone"
-  />
-</p>
+### Solve Problems
+
+It's easy to fall in the trap of "adding everything" - we explicitly don't want to do that. Everything added to `create-t3-app` should solve a specific problem that exists within the core technologies included. This means we won't add things like state libraries (`zustand`, `redux`) but we will add things like NextAuth.js and integrate Prisma and tRPC for you.
```
Comment (AyanavaKarmakar) [inline www/src/pages/en/introduction.md:42]:
```suggestion
It's easy to fall into the trap of "adding everything" - we explicitly don't want to do that. Everything added to `create-t3-app` should solve a specific problem that exists within the core technologies included. This means we won't add things like state libraries (`zustand`, `redux`) but we will add things like NextAuth.js and integrate Prisma and tRPC for you.
```

File: README.md [hunk 5]
```diff
@@ -50,17 +50,25 @@ The _"T3 Stack"_ is a web development stack made by [Theo](https://twitter.com/t
...
+
+We'll be frank - this is an _opinionated project_. We share a handful of core beliefs around building and we treat them as the basis for our decisions.
 
-We don't add libraries that are as simple as `npm install zustand`: **_If you cut an issue asking us to add your preferred libraries, we will make fun of you._**
+### 1. Solve Problems
 
-<p align="center">
-  <img src="https://media4.giphy.com/media/3orieLHXgpfkKO9Iju/200.gif" alt="Nelson from the Simpsons pointing and saying ha-ha in a mocking tone" />
-</p>
+It's easy to fall in the trap of "adding everything" - we explicitly _don't_ want to do that. Everything added to `create-t3-app` should solve a _specific_ problem that exists within the core technologies included. This means we **won't** add things like state libraries (`zustand`, `redux`) but we **will** add things like NextAuth.js and integrate Prisma and tRPC for you.
```
Comment (AyanavaKarmakar) [inline README.md:63]:
```suggestion
It's easy to fall into the trap of "adding everything" - we explicitly _don't_ want to do that. Everything added to `create-t3-app` should solve a _specific_ problem that exists within the core technologies included. This means we **won't** add things like state libraries (`zustand`, `redux`) but we **will** add things like NextAuth.js and integrate Prisma and tRPC for you.
```

**Review by AyanavaKarmakar — CHANGES_REQUESTED** [review]

**Review by juliusmarminge — APPROVED** [review]

File: www/src/pages/en/introduction.md (hunk 3 above)
Comment (c-ehrlich) [reply www/src/pages/en/introduction.md]: added this in #0fbc08c

**Review by nexxeln — APPROVED** [review]

**Pull request merged**